syntax = "proto3";

message Sensor {
    optional bool state = 1;
    optional float temperature = 2;
}

message SensorMsg {
    optional string operation = 1;
}
